packages/lablgtk/lablgtk.2.18.13/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "garrigue@math.nagoya-u.ac.jp"
+authors: ["Jacques Garrigue et al., Nagoya University"]
+homepage: "https://garrigue.github.io/lablgtk/"
+bug-reports: "https://github.com/garrigue/lablgtk/issues"
+dev-repo: "git+https://github.com/garrigue/lablgtk.git"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+build: [
+  ["./configure" "--prefix" prefix "LABLGLDIR=%{lib}%/lablgl"]
+  [make "world"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "4.06"}
+  "ocamlfind" {>= "1.2.1"}
+  "conf-gtk2" {build}
+  "camlp-streams" {build}
+]
+depopts: [
+  "conf-gtksourceview"
+  "conf-gnomecanvas"
+  "conf-glade"
+  "lablgl"
+]
+conflicts: [
+  "base-domains"
+]
```

Comment:
If this release is really adding OCaml 5.0 support then this should be safe to remove
```suggestion
```

Reply:
Thank you for the fix. Done.